src/Mesh/babylon.mesh.vertexData.ts

```diff
@@ -2095,7 +2165,7 @@
             }
 
             // facetPartitioning reinit if needed
-            if (computeFacetPartitioning) {  

```

Comment:
yaw sorry.. still getting used to VSCode, I did a format .. fixed in the coming update :)